Lib/pty.py

```diff
@@ -170,3 +175,128 @@ def spawn(argv, master_read=_read, stdin_read=_read):
 
     os.close(master_fd)
     return os.waitpid(pid, 0)[1]
+
+# Author: Soumendra Ganguly.
+SIGWINCH = signal.SIGWINCH
+
```

Comment:
"_login_pty" is based on the glibc implementation of login_tty(3). See
1. https://www.man7.org/linux/man-pages/man3/login_tty.3.html
2. https://sourceware.org/git/?p=glibc.git;a=blob;f=login/login_tty.c